ProjectLighthouse.Servers.GameServer/Controllers/CommentController.cs

```diff
@@ -119,7 +119,9 @@ public async Task<IActionResult> PostComment(string? username, string? slotType,
             targetId = await this.database.UserIdFromUsername(username!);
         }
 
-        bool success = await this.database.PostComment(token.UserId, targetId, type, comment.Message);
+        string scannedText = CensorHelper.ScanMessage(comment.Message);
```

Comment:
```suggestion
        string filteredText = CensorHelper.ScanMessage(comment.Message);
```
I think the scanned verb sounds a little weird since we're actually changing the message here

Reply:
I agree, I was mainly trying to keep it consistent with the existing naming convention in the message filtering controller. Change both maybe?